cuda_core/tests/test_module.py

```diff
@@ -180,6 +180,242 @@ def test_object_code_handle(get_saxpy_object_code):
     assert mod.handle is not None
 
 
+@pytest.fixture(scope="function")
+def get_ltoir_object_code(init_cuda):
```

Comment:
Please name this fixture following the convention `get_saxpy_kernel_ltoir` and move it closer to the other fixtures above